src/test/java/com/microsoft/sqlserver/jdbc/TestUtils.java

```diff
@@ -237,6 +237,15 @@ public static boolean isAzureMI(Connection con) {
         return ((SQLServerConnection) con).isAzureMI();
     }
 
+    /**
+     * Checks if connection is established to Azure Synapse OnDemand server
+     * 
+     */
+    public static boolean isAzureSynapseOndemand(Connection con) {
```

Comment:
Plus change name where it is called - TimeoutTest::273
```suggestion
    public static boolean isAzureSynapseOnDemand(Connection con) {
```